### PR TITLE
Hot Deals Section Tests

### PR DESCRIPTION
**Pull request by Vladislav2498**

### CODE REVIEW

File: client/src/pages/HomePage/HomePage.js [hunk 1]
```diff
@@ -1,114 +1,114 @@
...
+                <Button sx={{mt: '60px', textTransform: 'uppercase'}} endIcon={<ArrowRightAltIcon/>}>
+                    to our tours
+                </Button>
+            </HeroContainer>
+        </Box>
+        <Box component="section">
+            <AboutUkraine/>
+        </Box>
+        <Box component="section">
+            {/*<CardContainer />*/}
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/pages/HomePage/HomePage.js]: this is not corrent.. please leave it as it is as i already wrote the homepage test and mocked this component.

File: client/src/components/CardContainer/CardContainer.test.js [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+
+const RouterWrapper = ({children}) => <MemoryRouter>{children}</MemoryRouter>;
 
 describe('Card Container Snapshot test', () => {
     test('should CardContainer match snapshot', () => {
-        const { asFragment } = render(<CardContainer />);
+        const {asFragment} = render(<CardContainer/>, {wrapper: RouterWrapper});
         expect(asFragment()).toMatchSnapshot();
     });
 });
```
Comment (Ahmed-Fahmy-Dev) [inline client/src/components/CardContainer/CardContainer.test.js]: please mock the MUI components of typography
you can try to mock Swiper too

**Review by Ahmed-Fahmy-Dev — COMMENTED** [review]
please check Eslint errors:

image

**Review by Ahmed-Fahmy-Dev — CHANGES_REQUESTED** [review]
not approved.    
CardItem.test.js  is not complete.   MUI components have to be mocked there too

**Comment by Ahmed-Fahmy-Dev** [comment]
ESlint errors fixed by myself.

**Review by Ahmed-Fahmy-Dev — APPROVED** [review]

**Review by Ahmed-Fahmy-Dev — APPROVED** [review]
approved. 

i added a hot fix by myself.. please check commit message for details

**Review by AltysS — APPROVED** [review]

**Pull request merged**